README.md

```diff
@@ -53,9 +53,71 @@ public class JaxrsApplication extends Application {
 Just define them as Spring beans, and they will be automatically registered.
 Notice that JAX-RS resources can be singleton or request scoped, while JAX-RS providers must be singletons.
 
+
 ### Further information
 See [How to use RESTEasy Spring Boot Starter](mds/USAGE.md).
 
+
+## Testing
+
+### Adding POM dependency and enable integration-tests
+
+Add the Maven dependency below to your Spring Boot application pom file.<br>
+
+``` xml
+<dependencies>
+    <dependency>
+        <groupId>com.paypal.springboot</groupId>
+        <artifactId>resteasy-spring-boot-test</artifactId>
+        <version>2.2.2-SNAPSHOT</version>
+        <scope>test</scope>
+    </dependency>
+</dependencies>
+
+<build>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <executions>
+                <execution>
+                    <id>failsafe-integration-tests</id>
+                    <phase>integration-test</phase>
+                    <goals>
+                        <goal>integration-test</goal>
+                        <goal>verify</goal>
+                    </goals>
+                </execution>
+            </executions>
+        </plugin>
+    </plugins>
+</build>
+
+```
+
+### Writing a test class
+
+Create a new class in `src/test/java` with a name ending with `IT`.
+
+``` java
+
+public class EchoIT extends RestEasyApplicationTest {
+    public EchoIT() {
+        // provide spring application main class
+        // and the base path to the application
+        super(Application.class, "/sample-app/echo");  
```

Comment:
The "base path" of the sample-app is just "/sample-app". The echo part in the URI comes from the Path annotation from the resource class.